apps/spruce/src/pages/spawn/spawnHost/EditSpawnHostModal.tsx

```diff
@@ -41,7 +50,12 @@ export const EditSpawnHostModal: React.FC<EditSpawnHostModalProps> = ({
     volumesData,
   } = useLoadFormData(host);
 
-  const instanceTypes = instanceTypesData?.instanceTypes ?? [];
+  let instanceTypes = instanceTypesData?.instanceTypes ?? [];
+  // The list of instance types provided by Evergreen can be out-of-date, so make sure the instance type in use is considered valid by RJSF
+  if (!instanceTypes.includes(host.instanceType)) {
+    instanceTypes = [...instanceTypes, host.instanceType];
+  }
```

Comment:
lol nice, could we also file a ticket for someone to take a look at these and update the list in admin settings later?